contrib/ansible/roles/common/tasks/system-proxy.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Proxy | Install system http proxy environment variable
+  lineinfile: dest=/etc/environment regexp=^HTTP_PROXY= line=HTTP_PROXY="{{ system_http_proxy }}"
```

Comment:
help me understand exactly which pieces this affect and which it doesn't....

(the rest of the PR looks good)